authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -250,7 +250,7 @@ func getDataActions(subRevReq *authzv1.SubjectAccessReviewSpec, clusterType stri
 
 		isCustomerResourceTypeCheckAvailable := allowCustomResourceTypeCheck && storedOperationsMap != nil && len(storedOperationsMap) != 0
 		if !isCustomerResourceTypeCheckAvailable {
-			klog.V(5).Info("CustomResource type verification is not available for this request")
+			klog.V(5).Info("CustomResourceTypeCheck feature is not available for this request", "allowCustomResourceTypeCheck", allowCustomResourceTypeCheck, "operationsMapAvailable", storedOperationsMap != nil && len(storedOperationsMap) != 0)
```

Comment:
here is why i like https://github.com/go-logr/logr more. you can create the logger with value at the beginning. then you don't need to repeat `CustomResourceTypeCheck` or `CustomResourceTypeCheck:` all over in this function